packages/safe-ds-vscode/src/extension/mainClient.ts

```diff
@@ -128,29 +128,70 @@ const acceptRunRequests = function (context: vscode.ExtensionContext) {
     }, 'runtime_error');
     // Register VS Code Entry Points
     context.subscriptions.push(
-        vscode.commands.registerCommand('extension.safe-ds.runPipelineFile', (filePath: vscode.Uri | undefined) => {
-            let pipelinePath = filePath;
-            // Allow execution via command menu
-            if (!pipelinePath && vscode.window.activeTextEditor) {
-                pipelinePath = vscode.window.activeTextEditor.document.uri;
-            }
-            if (
-                pipelinePath &&
-                !services.LanguageMetaData.fileExtensions.some((extension: string) =>
-                    pipelinePath!.fsPath.endsWith(extension),
-                )
-            ) {
-                vscode.window.showErrorMessage(`Could not run ${pipelinePath!.fsPath} as it is not a Safe-DS file`);
-                return;
-            }
-            if (!pipelinePath) {
-                vscode.window.showErrorMessage('Could not run Safe-DS Pipeline, as no pipeline is currently selected.');
-                return;
-            }
-            const pipelineId = crypto.randomUUID();
-            printOutputMessage(`Launching Pipeline (${pipelineId}): ${pipelinePath}`);
-            executePipeline(services, pipelinePath.fsPath, pipelineId);
-        }),
+        vscode.commands.registerCommand(
```

Comment:
The function `acceptRunRequests` gets quite long, and it does more than its name implies. Better split this into three separate ones:
1. Register message callbacks
2. Register VS Code commands (maybe even extract the callback for each command into a named function)
3. Register watchers

Later we can move parts of this into separate files.